src/java.base/share/classes/java/net/SocketPermission.java

```diff
@@ -713,14 +706,14 @@ private boolean match(String cname, String hname) {
 
     private boolean authorized(String cname, byte[] addr) {
```

Comment:
now `cname` parameter is unused in this method

Reply:
`cname` has been deleted.